compact/ecal/barrel_interlayers.xml

```diff
@@ -68,7 +69,7 @@
       Adjusting size of the ScFi back chunk to match number of imiaging layers
       and 20 radiation lengths in total
```

Comment:
```suggestion
      Adjusting size of the ScFi back chunk to match number of imaging layers
      and `EcalBarrel_TotalFiberLayers_num` radiation lengths in total
```

Reply:
Thanks Dmitry for catching the typo. The Calo is still ~20 radiation length in in total not `EcalBarrel_TotalFiberLayers_num` (16) radiation length in total (the total sum of SciFi/Pb calo is still the same). The SciFi "layers" (that define radial size of readout) are now 16*1.22 mm = 19.52 mm as opposed to 13*1.22 mm = 15.86 mm in the previous version, but the total length is still ~32 cm which is ~20 X0.